### ISSUE TITLE
[Accessibility] Slides Page: Inaccessible button symbol and instruction for slide preview

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#993]
**Describe the bug**
To upload a slide, users have to follow a set of instructions. The third step says to "Click on the (arrow symbol) button to get check a preview of the slide". This instruction relies solely on shape and color to understand the content and so fails the sensory characteristics criterion of the WCAG standards. Plus, the arrow symbol cannot be read by a screen reader.

**To Reproduce**
1. Go to 'Slides' on the nav bar
2. Click on 'upload' button

**Expected behavior**
Replace the arrow symbol with a descriptive text label that conveys the action. For example, instead of relying on just the arrow symbol, use "Preview Slide" or "Check Slide Preview" as the label.

**Screenshots**
![image](https://github.com/camicroscope/caMicroscope/assets/122698422/423a6146-5ef0-4172-b7a3-df0061dbd6c6)

![image](https://github.com/camicroscope/caMicroscope/assets/122698422/da822ed5-b422-4404-b60a-c654629ee640)
